cypress/integration/dictionary/createVersion.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Creating a dictionary version Releasing it and copy subscription URL
+  Background:
+    Given the user is logged in
+
+  @dictionary
+  Scenario: The user should be able to click the button to create a new version
+    Given a dictionary exists
+      And the user is on the dictionary page
+    When the user clicks the create new version button
+    Then the user should be on the create new version dialog box
+
+  @dictionary
+  @version
+  Scenario: The user should be able to create a new version, release it and copy subscription URL
```

Comment:
I would actually break this out as three separate tests: one to create a new version. One to release it via the checkbox, and one to copy the subscription URL. That way we should also be able to check that the user can release a new version within the "Create New Version" form.

Reply:
You are right @ibacher that's what @suruchee had but the last two scenarios were not passing probably because they needed a version to be created, she had a command for creating and getting a version the way the dictionary commands are but they were not working, so I told her that we first put those other two tests in the same scenario so that we see they pass, then we reach out to you about separating them.

Reply:
Fair enough... The version release process actually does some backend processing, so between the request to create the version returning and the version actually being created there's a bit of a delay... not much for a small dictionary, but not no time at all either.

Probably the simplest way to do this is to force Cypress to pause immediately after submitting the new version request, e.g., by using `cy.wait(500)`. A more complicated version of the same idea would be to have a shorter pause and at the end of each pause poll the backend until a longer period of time, e.g. 8 seconds or w/e had passed.

Reply:
Here's a library to handle my "more complicated" version: https://github.com/NoriSte/cypress-wait-until. Basically looks like you can use that and wait until the backend returns that the version exists. Note that it's probably also necessary to force Cypress to reload the page.

Reply:
Thank you @hadijahkyampeire  and @ibacher, I will update with the delay suggestion.